app/main.py

```diff
@@ -1,3 +1,45 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f'Distance: {self.km} kilometers.'
+
+    def __repr__(self):
+        return f'Distance(km={self.km})'
+
+    def __add__(self, other):
+        if isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, (int, float)):
+            self.km += other
+        else:
```

Comment:
 You don’t need `else` if you have a `return` statement in `if`. Rewrite without `else`